src/sa_web/static/js/views/place-detail-view.js

```diff
@@ -60,6 +60,11 @@ var Shareabouts = Shareabouts || {};
 
       // Augment the template data with the attachments list
       data.attachments = this.model.attachmentCollection.toJSON();
+      var i, a;
+      for (i = 0; i < data.attachments.length; i++) {
+        a = data.attachments[i];
+        a.ratio = 100.0 * a.height / a.width;
+      }
```

Comment:
@yuhengd3 What would you think about moving this into a separate function -- something like:

```javascript
annotateWithAspectRatios: function(attachments) {
  var i, a;
  for (i = 0; i < data.attachments.length; i++) {
    a = data.attachments[i];
    a.ratio = 100.0 * a.height / a.width;
  }
},
```

Then from we could just call that from here:
```javascript
this.annotateWithAspectRatios(data.attachments);
```